StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Components/Notice/LinkNoticeView.swift

```diff
@@ -82,7 +82,6 @@ final class LinkNoticeView: UIView {
         addAndPinSubview(stackView)
 
         backgroundColor = type.backgroundColor
-        layer.cornerRadius = LinkUI.mediumCornerRadius
```

Comment:
This view has the same background color as the container, so we keeping corners sharp is fine. (Mostly removed it because we no longer use `mediumCornerRadius`).